test/testcafe/spec/NumberChallengePushView_spec.js

```diff
@@ -59,6 +59,7 @@ test
   });
 
 test
+  .meta('v3', false) // OKTA-587189 - disabled in v3 due to immediate polling issue
```

Comment:
we have coverage for this in integration tests so I dont think it will be a problem to disable this for now

Reply:
I am curious now, will this work now with the latest changes you made in the 429 polling branch?

Reply:
unfortunately this one still does not work ☹️